src/keywords/speciesvector.cpp

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2021 Team Dissolve and contributors
+
+#include "keywords/speciesvector.h"
+#include "base/lineparser.h"
+#include "classes/coredata.h"
+#include "classes/species.h"
+
+SpeciesVectorKeyword::SpeciesVectorKeyword(std::vector<const Species *> references)
+    : KeywordData<std::vector<const Species *>>(KeywordBase::SpeciesVectorData, references)
```

Comment:
Think we can potentially make a move here to save one copy of our vector
```suggestion
    : KeywordData<std::vector<const Species *>>(KeywordBase::SpeciesVectorData, std::move(references))
```